packages/ecl-fractal-theme/src/search.js

```diff
@@ -22,6 +23,9 @@ module.exports = (theme, env, app) => {
       handle: c.handle,
       title: c.title,
       notes: c.notes ? `${c.notes.substring(0, 50)} ...` : '',
+      variants: c.variants.items.length
```

Comment:
Just to be sure, I'd do `c.variants && c.variants.items && c.variants.items.length ?`, but if it works like it that, I'm okay with it